docs/reference/commandline/cli.md

```diff
@@ -78,6 +78,11 @@ For example:
 Instructs Docker to use the configuration files in your `~/testconfigs/`
 directory when running the `ps` command.
 
+> **Note:**
+> Calling docker with `sudo` results in Docker looking first in `/root/.docker/`.
+> If it isn't found it looks for the user who did the `sudo` call and resolves the
```

Comment:
Hum, I like the *"the user that did the sudo call"* in fact because it's not prone to interpretation. At first, current user for me means the one that did the sudo call, but I can see some confusion basde on your comment @runcom. So I'm fine with the current sentences. :stuck_out_tongue_closed_eyes: